src/views/RecIM/components/Forms/ParticipantForm/index.jsx

```diff
@@ -0,0 +1,122 @@
+import { useState, useMemo } from 'react';
+import Form from '../Form';
+import { createCustomParticipant, editCustomParticipant } from 'services/recim/participant';
+
+const genderTypes = ['U', 'M', 'F'];
+
+const createParticipantFields = [
+  {
+    label: 'First Name',
+    name: 'FirstName',
+    type: 'text',
+    helperText: '*Required',
+    required: true,
+  },
+  {
+    label: 'Last Name',
+    name: 'LastName',
+    type: 'text',
+    helperText: '*Required',
+    required: true,
+  },
+  {
+    label: 'Specified Gender',
+    name: 'SpecifiedGender',
+    type: 'select',
+    menuItems: genderTypes.map((type) => type),
+  },
+  {
+    label: 'Turn On Email Notifications',
+    name: 'AllowEmails',
+    type: 'checkbox',
+    helperText: '*Required', // checkbox: either 0 or 1 => default 1
+    required: false,
+  },
+];
+
+const ParticipantForm = ({
+  participant,
+  createSnackbar,
+  onClose,
+  openParticipantForm,
+  setOpenParticipantForm,
+}) => {
+  const [isSaving, setSaving] = useState(false);
+
+  const currentInfo = useMemo(() => {
+    if (participant) {
+      return {
+        FirstName: participant.FirstName,
+        LastName: participant.LastName,
+        SpecifiedGender: participant.SpecifiedGender,
+        AllowEmails: participant.AllowEmails,
+      };
+    }
+    return {
+      FirstName: '',
+      LastName: '',
+      SpecifiedGender: 'U',
+      AllowEmails: true,
```

Comment:
We have an "AllowEmails" but we don't have a way of storing emails of non-gordon participants. I think we either need another column for emails OR you need to set it always to false.